hikari/impl/stateful_cache.py

```diff
@@ -1598,3 +1605,27 @@ def update_voice_state(
         cached_voice_state = self.get_voice_state(voice_state.guild_id, voice_state.user_id)
         self.set_voice_state(voice_state)
         return cached_voice_state, self.get_voice_state(voice_state.guild_id, voice_state.user_id)
+
+    def delete_message(self, message_id: snowflakes.Snowflake) -> None:
+        if message_id in self._message_entries:
+            self._message_entries.pop(message_id)
+
+    def delete_messages(self, message_ids: typing.AbstractSet[snowflakes.Snowflake]) -> None:
```

Comment:
is there any benefit to having a special case multi-delete method here rather than just looping through and deleting each one specifically in the place where this is being called?